### DISCUSSION TITLE
class color friendly names without healthbar

### DISCUSSION THREAD

**Discussion by elmrok** [siweia/NDui#137] [Q&A] [answered]
Is there a way to set up the names , including your own, to be in class color without any healthbar? I have been using the Name only mode when friendly healthbars are active but they still turn on in raids. The bigger and colored text is much easier to read and keep up with who is where but when it adds in health bars in raids it gets full on screen. It would also be nice to have my own name in my class color.

**Comment by siweia** [comment, marked as answer]
Nameplates in dungeons or raid are restricted by blizzard. There are no addons can do the thing you mentioned.

**Reply by elmrok** [reply]
ah okay, did not know that. thank you
